testing/testcafe/tests/dataGrid/focusedRow/markup.ts

```diff
@@ -0,0 +1,68 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { changeTheme } from '../../../helpers/changeTheme';
+import createWidget from '../../../helpers/createWidget';
+import url from '../../../helpers/getPageUrl';
+import { testScreenshot } from '../../../helpers/themeUtils';
+import DataGrid from '../../../model/dataGrid';
+import { Themes } from '../helpers/themes';
+
+fixture.disablePageReloads`Focused row - markup`
+  .page(url(__dirname, '../../container.html'));
+
+// TODO: Enable multi-theming testcafe run in the future.
+[
+  Themes.genericLight,
+  Themes.materialBlue,
+].forEach((theme) => {
+  test(`markup - ${theme}`, async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+    const dataGrid = new DataGrid('#container');
+    const firstCell = dataGrid.getDataCell(0, 0);
+    const secondCell = dataGrid.getDataCell(0, 1);
+    const thirdCell = dataGrid.getDataCell(0, 2);
+
+    await t.click(firstCell.element);
+    await t.typeText(firstCell.getEditor().element, 'TEST', { replace: true });
+
+    await t.click(secondCell.element);
+    await t.typeText(secondCell.getEditor().element, ' ', { replace: true });
+
+    await t.click(thirdCell.element);
+
+    await testScreenshot(t, takeScreenshot, `focused-row_markup ${theme}.png`, { element: dataGrid.element });
```

Comment:
minor: let's rename file as `testScreenshot` would name it, so we won't need to rename files after enabling multi-theming tests

Reply:
Discussed and decided to replace it with default ```takeScreenshot``` while we didn't enable the multi-theming tests